src/index.js

```diff
@@ -1,46 +1,25 @@
-import { ERROR_CODES } from "./constants";
 import { Race } from "./domain";
 import { RandomMoveStrategy } from "./domain/strategies";
 import { View } from "./views";
 
 export class App {
-  #race;
-
   async play() {
     try {
-      const carNames = await this.#getCarNames();
-      if (carNames.length === 0) return;
-
-      this.#race = new Race(carNames, new RandomMoveStrategy());
-      const result = this.#race.race();
-      this.#render(result);
+      const carNames = await View.getCarNamesPrompt();
+      const raceRound = await View.getRaceRoundPrompt();
+
+      const race = new Race(carNames, raceRound);
+      const raceResult = race.race(new RandomMoveStrategy());
+      View.printRoundStart();
+      raceResult.forEach((result) => {
+        View.printRaceResult(result.cars);
+      });
```

Comment:
```suggestion
    View.printRaceResult(result);
```

한 번에 출력할 수도 있을 것 같아요!